src/components/CreatePost/PostDetail.tsx

```diff
@@ -1,42 +1,91 @@
-import { OutlinedInput, Stack } from '@mui/material';
+import { AddCircle } from '@mui/icons-material';
```

Comment:
proposal;
```suggestion
import { AddCircle as AddCircleIcon } from '@mui/icons-material';
```

`@mui/icons-material` 의 아이콘 컴포넌트라고 명시적으로 알려주기 위해서 컴포넌트 이름 뒤에 Icon을 붙이면 좋겠어요.

Reply:
음 다른 분의 코드를 볼 때 이게 아이콘인지 직접 만든 컴포넌트인지 헷갈렸던 경험이 있습니다. 좋은 의견 같습니다.